utilities/ubi8-asciidoctor/Dockerfile

```diff
@@ -39,11 +39,12 @@ RUN microdnf install \
   patch \
   git \
   gcc-c++ \
-  glib2-devel
+  glib2-devel \
+  && microdnf clean all
 
 # install pandoc
 RUN curl -L https://github.com/jgm/pandoc/releases/download/${PANDOC_VERSION}/pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz -o pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz
-RUN tar xvzf pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz --strip-components 1 -C /usr/local/
+RUN tar xvzf pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz --strip-components 1 -C /usr/local/ 
```

Comment:
Sorry @buuhsmead, I missed this trailing space the first time

Reply:
No Problem, have to adjust several editors to remove at least spaces at the end of a line.